src/Raven.Studio/typescript/viewmodels/shell.ts

```diff
@@ -258,25 +259,24 @@ class shell extends viewModelBase {
 
         if (this.clientCertificate() && this.clientCertificate().Name) {
             
-            const dbAccess = certificateModel.resolveDatabasesAccess(this.clientCertificate())
-                .map(x => `<div>${x}</div>`)
-                .join("");
-            
+            const dbAccess = certificateModel.resolveDatabasesAccess(this.clientCertificate());
```

Comment:
update this method to never return null.

Reply:
done